src/domain/accounts/index.types.d.ts

```diff
@@ -119,6 +119,10 @@ interface IAccountsRepository {
   listUnlockedAccounts(): Promise<Account[] | RepositoryError>
   findById(accountId: AccountId): Promise<Account | RepositoryError>
   findByUserId(userId: UserId): Promise<Account | RepositoryError>
+
+  findByKratosUserId(kratosUserId: KratosUserId): Promise<Account | RepositoryError>
+  persistNewKratosUser(kratosUserId: KratosUserId): Promise<Account | RepositoryError>
```

Comment:
```suggestion
  persistNewByKratosUserId(kratosUserId: KratosUserId): Promise<Account | RepositoryError>
```

Reply:
this method will be removed in a future PR and will only be called `persistNew`, where the kratosUserId will be a mandatory parameters (alongside other parameters currently passed for phone authentication)